### DISCUSSION TITLE
Maximum supported amount of buttons and relays

### DISCUSSION THREAD

**Discussion by sanchosk** [arendst/Tasmota#10378] [Q&A] [answered]
Hi, all.
I've purchased a smart socket with 4 outputs. I opted for specifically this one as I wanted to have one with button for each separate output.
https://www.aliexpress.com/item/32889240989.html?spm=a2g0s.9042311.0.0.27424c4dUD4j0D
The relays are controlled by ESP outputs, but the buttons are going to some onboard chip - Tuya protocol seemed not to work for it, so I simply de-soldered the chip and used coil wire to connect the ESP outputs to the specific buttons.
Everything works OK, but I have minor issue.
The socket has 4 outputs, but also a additional blue LED and additional "all on/off" button.
I would like to define the LED as specific output (led 5 or relay 5) and the button as button #5.
However, the GUI only allows me to select 4 relays and 4 buttons.

Is there any way how I can define 5 buttons and set custom action of the 5th button?

**Comment by barbudor** [comment, marked as answer]
I believe we are already dealing with your question on Discord ?
